### PR TITLE
Update CVC Recollection API

### PR DESCRIPTION
**Pull request by wooj-stripe**
## Summary
Updates API for CVC Recollection & removes beta flag

## Motivation
CVC Recollection GA

## Testing
early access tested, existing tests

## Changelog
Updated

### CODE REVIEW

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift [hunk 1]
```diff
@@ -66,20 +62,20 @@ public extension PaymentSheet {
...
+                    requireCVCRecollection: Bool = false) {
             self.mode = mode
             self.paymentMethodTypes = paymentMethodTypes
             self.onBehalfOf = onBehalfOf
             self.paymentMethodConfigurationId = paymentMethodConfigurationId
             self.confirmHandler = confirmHandler
-            self.isCVCRecollectionEnabledCallback = isCVCRecollectionEnabledCallback ?? { return false }
+            self.requireCVCRecollection = requireCVCRecollection
             validate()
         }
```
Comment (yuki-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift:80]: We can delete this once we remove `EarlyAccessCVCRecollectionFeature` right?

File: Example/PaymentSheet Example/PaymentSheet Example/ExampleSwiftUICustomPaymentFlowCVCRecollection.swift [hunk 2]
```diff
@@ -137,6 +144,19 @@ class MyCustomBackendCVCRecollectionModel: ObservableObject {
             })
         task.resume()
     }
+    func intentConfiguration() -> PaymentSheet.IntentConfiguration {
```
Comment (yuki-stripe) [inline Example/PaymentSheet Example/PaymentSheet Example/ExampleSwiftUICustomPaymentFlowCVCRecollection.swift:147]: If we move this functionality (ability to _update_ FlowController w/ new IntentConfiguration) to the playground, can we delete this file?

**Review by yuki-stripe — APPROVED** [review]

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift (hunk 1 above)
Comment (wooj-stripe) [reply StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift:80]: yep!

File: Example/PaymentSheet Example/PaymentSheet Example/ExampleSwiftUICustomPaymentFlowCVCRecollection.swift (hunk 2 above)
Comment (wooj-stripe) [reply Example/PaymentSheet Example/PaymentSheet Example/ExampleSwiftUICustomPaymentFlowCVCRecollection.swift:147]: Yep!

**Pull request merged**